modules/prebidServerBidAdapter/index.js

```diff
@@ -133,6 +133,13 @@ function formatUrlParams(option) {
       let temp = option[prop];
       option[prop] = { p1Consent: temp, noP1Consent: temp };
     }
+    if (utils.isPlainObject(option[prop]) && (!option[prop].p1Consent || !option[prop].noP1Consent)) {
```

Comment:
I think this can be simplified to
```
if (utils.isPlainObject(option[prop]))
```
The other conditions are being checked again within the `forEach`

Reply:
That's not the same.  

The `forEach` above this code is handling the condition when `endpoint: 'someURLstring'`.  The if-check here checks if the `endpoint` is an object config and it's missing either of the properties `p1Consent` or `noP1Consent`.

Reply:
Sorry, I meant the `forEach` within the `if` statement. It checks if either of those properties is missing again.

Reply:
oh - I see what you meant.  I was hoping to avoid this forEach if the config was setup 'correctly' - that's why I had them included in the outer if-check.